docs/dev/testing.md

```diff
@@ -0,0 +1,40 @@
+# Running the tests
+
+## Running all tests
+
+The tests for all languages may be run from the command line:
+
+```
+make test
+```
+
+---
+
+_Windows Note:_ If `make` is not installed, it may be obtained from the [MozillaBuild shell](https://wiki.mozilla.org/MozillaBuild) or by using the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/faq).
```

Comment:
Let's leave the MozillaBuild shell out for now, I'd rather not advise people to do so, if possible.